packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -159,6 +174,13 @@ export type AutocompleteProps<
    * You will need to implement this function if your `option` items are objects.
    */
   getIsOptionEqualToValue?: (option: OptionType, value: OptionType) => boolean;
+
+  /**
+   * If this component is required to display a virtualized list of options, this value needs
+   * to be toggled to true.
+   * It is recommended if you're options are on the order of 10's of hundreds or more
+   */
+  isVirtualized?: boolean;
```

Comment:
Can `isVirtualized` be changed after loading the component or is it a one-and-done kinda thing like with our table component?

In those cases, I think they should probably be separate components because the assumption in React is that props can be manipulated at runtime.

Although, I think we don't do that on our Table right now, but Table is also in Labs still.

Reply:
I added a `useRef` here to capture the initial value of `isVirtualized` to address this.

The more I think about it though, we may want to allow this prop to be toggled? I believe if a user initially populates this with 10's of rows, they may want this off, but if they're paging data and end up with 100's or 1000's of rows, they may want to toggle this on then to make it more performant?

Reply:
If someone is turning on the `virtualized` prop, they probably already know why they're doing this. I don't believe they should be toggling this on and off.